packages/wonder-blocks-accordion/src/components/accordion.tsx

```diff
@@ -60,6 +60,15 @@ type Props = AriaProps & {
      * value is prioritized.
      */
     cornerKind?: AccordionCornerKindType;
+    /**
+     * Whether to include animation on the header. This should be false
+     * if the user has `prefers-reduced-motion` opted in. Defaults to false.
+     *
+     * If this prop is specified both here in the Accordion and within
+     * a child AccordionSection component, the Accordion’s animated
```

Comment:
I actually think we're gonna change all these props so that the AccordionSection's props are prioritized. I'll add this in the following PR.